supervisor/utils/log_format.py

```diff
@@ -1,15 +1,21 @@
 """Custom log messages."""
 import re
 
+import sentry_sdk
+
+from ..exceptions import APIError
+
 RE_BIND_FAILED = re.compile(r".*Bind for.*:(\d*) failed: port is already allocated.*")
 
 
 def format_message(message: str) -> str:
     """Return a formated message if it's known."""
-    match = RE_BIND_FAILED.match(message)
-    if match:
-        return (
-            f"Port '{match.group(1)}' is already in use by something else on the host."
-        )
+    try:
+        match = RE_BIND_FAILED.match(message)
+        if match:
+            return f"Port '{match.group(1)}' is already in use by something else on the host."
+    except TypeError as err:
+        sentry_sdk.capture_exception(err)
+        raise APIError("Type of message is not string") from None
```

Comment:
We can't raise an APIError because we use that on the API response level

Reply:
Would `HassioError` be better?